### ISSUE TITLE
RDflow - přidání sloupce Úkol

### ISSUE THREAD

**Issue by nezbedova** [proarc/proarc#1308]
Stále bojuji s Úkoly. Nemohu si vzpomenout jak to mělo fungovat.

Proto bych chtěla vědět, zda by bylo technicky možné, aby se ve Správě záměrů zobrazoval sloupec Úkoly, který by vycházel z prostřední tabulky vpravo. Ukazovat by se tam poslední dokončený úkol daného záměru.

**Comment by SykoraLukas** [comment]
Ano, tohle by asi šlo doplnil. Jen poslední dokončený v čase nebo v pořadí úkolu? Příklad: Popisu objekt, vyexportuji K4 a dokončí se mi K4 export. Pak si teprve vzpomenu, ze jsem nedokončil úkol “Popis v ProArc”. Doplním to do úkolu a teď, jaký úkol má být napsaný v záměru - k4 export, protože je v pořadí úkolu až za popisem, nebo popis, protože byl dokončen jako poslední?

**Comment by nezbedova** [comment]
Poslední ze seznamu. Pokud někdo doplní dokončení dodatečně, tak se to neprojeví. Z Vašeho příkladu by to byl k4 export. Dalo by se tak sledovat jak daleko s digitalizací určitého titulu jsme. U každého záměru bych viděla, co je v jaké fázi. S lidskou chybou počítám.

**Comment by nezbedova** [comment]
Ještě mě napadlo, zda by tam nešel přidat i sloupec Změněno z úkolů. Že by bylo vidět, kdy byl ten poslední úkol dokončený. Ale to už je asi moc.

**Comment by SykoraLukas** [comment]
Zjistím, jaké jsou možnosti. Ale teoreticky by to mělo jít.

**Comment by nezbedova** [comment]
Otestováno. Funguje, pokud nepřidám úkol z rozbalovátka

**Comment by nezbedova** [comment]
Prosím ještě o přidání sloupce _Uživatel_ z úkolů. Teď tam je Uživatel ze záměru. Toho nepotřebuji. Ale mohl by se hodit někomu jinému. Zadám issue na přejmenování, aby bylo jasné, který je který.

**Comment by SykoraLukas** [comment]
Na schůzce 2021-06-23 bylo dohodnuto, že uživatel, který vytvořil záměr je zbytečný. Místo něj musí být uživatel, který dokončil poslední úkol.

**Comment by nezbedova** [comment]
Otestováno.
Navrhuji zavřít.

**Comment by nezbedova** [comment]
Export pro Kramerius se zapíše opravdu až po exportu. Pokud je chyba - nezapíše se. To je správně.

**Comment by SykoraLukas** [comment]
Uzavírám na základě labelu.
